opennms-base-assembly/src/main/filtered/etc/service-configuration.xml

```diff
@@ -172,15 +172,7 @@ maintained
   </service>
   <service>
     <name>OpenNMS:Name=Provisiond</name>
-    <class-name>org.opennms.netmgt.daemon.SimpleSpringContextJmxServiceDaemon</class-name>
-    <attribute>
-      <name>LoggingPrefix</name>
-      <value type="java.lang.String">provisiond</value>
-    </attribute>
-    <attribute>
-      <name>SpringContext</name>
-      <value type="java.lang.String">provisiondContext</value>
-    </attribute>
+    <class-name>org.opennms.netmgt.provision.service.jmx.Provisiond</class-name>
```

Comment:
It should work without the configuration change, but spring did not initialize the beans then and I was not able to figure out why. So yeah, it is required (-: